src/v/cloud_roles/refresh_credentials.h

```diff
@@ -188,17 +193,17 @@ refresh_credentials make_refresh_credentials(
         host = cfg_host.value();
     }
     auto port = endpoint ? endpoint->port() : CredentialsProvider::default_port;
+    net::unresolved_address address{{host.data(), host.size()}, port};
+    vlog(
```

Comment:
nit: info-level messages should be phrased in terms the user could  understand, perhaps something like "Using credential service at {}"

Reply:
I have removed the log, as soon after this statement the address is printed again via `remote.cc`